includes/cf/sonarr-anime.md

```diff
@@ -22,7 +22,7 @@
     | [{{ sonarr['cf']['anime-web-tier-06-fansubs']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#anime-web-tier-06-fansubs)                   |       {{ sonarr['cf']['anime-web-tier-06-fansubs']['trash_scores']['default'] }}        | {{ sonarr['cf']['anime-web-tier-06-fansubs']['trash_id'] }}          |
     | [{{ sonarr['cf']['anime-raws']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#anime-raws)                                                 |               {{ sonarr['cf']['anime-raws']['trash_scores']['default'] }}               | {{ sonarr['cf']['anime-raws']['trash_id'] }}                         |
     | [{{ sonarr['cf']['anime-lq-groups']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#anime-lq-groups)                                       |            {{ sonarr['cf']['anime-lq-groups']['trash_scores']['default'] }}             | {{ sonarr['cf']['anime-lq-groups']['trash_id'] }}                    |
-    | [{{ sonarr['cf']['av1']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#av1)                                                               |                                         -10000                                          | {{ sonarr['cf']['av1']['trash_id'] }}                                |
+    | [{{ sonarr['cf']['av1']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#av1)                                                               |                  {{ sonarr['cf']['av1']['trash_scores']['default'] }}                   | {{ sonarr['cf']['av1']['trash_id'] }}                                |
```

Comment:
```suggestion
    | [{{ sonarr['cf']['av1']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#av1)                                                               |                  {{ sonarr['cf']['av1']['trash_scores']['anime-sonarr'] }}                   | {{ sonarr['cf']['av1']['trash_id'] }}                                |
```

Reply:
Might need the table formatting tidying here too.

Reply:
of course. looks okay my computer.

![image](https://github.com/TRaSH-Guides/Guides/assets/3604363/f93cd28e-96a6-4eef-bfef-3c6ffab7b3e2)